### PR TITLE
Added a copyright disclaimer on all the src files

### PR DESCRIPTION
**Pull request by mosekwa**

### CODE REVIEW

**Comment by mheki** [comment]
You can also add phpspec class templates for new classes. Example: https://github.com/phpspec/phpspec/blob/master/.phpspec/class.tpl

File: .idea/encodings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pierredup) [inline .idea/encodings.xml]: The `.idea` folder should not be committed

File: composer.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "php": "^7.0",
     "doctrine/collections": "~1.3",
     "http-interop/http-factory": "^0.2.0",
+    "phpro/soap-client": "dev-master",
```
Comment (pierredup) [inline composer.json]: This package should not depend on itself

File: bin/soap-client [hunk 3]
```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env php
 <?php
 
+/*
+ * This file is part of the Phpro application.
```
Comment (pierredup) [inline bin/soap-client]: Should the name possibly be `Phpro SoapClient`, since phpro is just the vendor name?

File: .idea/encodings.xml (hunk 1 above)
Comment (mosekwa) [reply .idea/encodings.xml]: Noted, thanks. I will remove it and also add it to the git ignore file.

**Comment by veewee** [comment]
We've dicussed this internally and decided not to add these copyright headers. Thanks for the PR anyways.

**Pull request closed without merging**